### ISSUE TITLE
* and info icon sometimes display in the Translation Identifier text field on the project details screen

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#4685]
I think this occurs with the * displaying only after tC startup. In sp or pt-br both the * and info icon show there (see second comment) even after the first time through.

![image](https://user-images.githubusercontent.com/8171759/42348374-d9f1afec-806e-11e8-9f3d-3e59be2f93d1.png)

![image](https://user-images.githubusercontent.com/8171759/42348386-e27c3056-806e-11e8-9650-339f70868c6b.png)

May want to combine with #4759

**Comment by cckozie** [comment]
This is worse in some other languages. The field header should never be wrapped into the entry portion of the field.
On import:
![image](https://user-images.githubusercontent.com/8171759/48428849-7bb9fb80-e731-11e8-95d4-3d4b0c06011c.png)

In Edit details:
![image](https://user-images.githubusercontent.com/8171759/48433927-f6891380-e73d-11e8-820a-ad9ca1162900.png)

**Comment by richmahn** [comment]
- [x] [4] Work on CSS for field labels for #4685
- [x] [1] PR Review for #4685

**Comment by cckozie** [comment]
I'm not seeing any change in 1.1.0 (0179053) 
![image](https://user-images.githubusercontent.com/8171759/49893348-c605c980-fe10-11e8-9046-d88f8dbc54e0.png)

**Comment by richmahn** [comment]
@cckozie Before that red asterisk would be covering the the text field. Since it is a span tag in html, it always wraps, but before you couldn't even see the text below the label.

**Comment by cckozie** [comment]
@richmahn A couple things: 
A) There must be something a little strange going on with this dialog since the problem only occurs the first time the dialog is displayed after starting up tC. As long as I don't restart, I won't see it again.
B) There is nothing in my past experiences with programming that helps me relate to how this can be unfixable or even very difficult to fix. I don't believe I have ever seen a problem like this in any other app I've ever used. If you don't think we should expend the effort to actually fix it, please help me understand.
C) Would a simple fix be to just make the form a few pixels wider by default? (There is another issue dealing with more significant problems when the form is shrunk)

**Comment by richmahn** [comment]
I actually changed how the whole layout works, but let me look at it again. And see if I can make it work a little better and use up the whole area without too much white space.

**Comment by cckozie** [comment]
I noticed today that there is another problem, when the language fields are empty, the horizontal alignment of the fields is now off too.
![image](https://user-images.githubusercontent.com/8171759/50181115-2b563080-02d1-11e9-9a92-7702c49972c2.png)

**Comment by richmahn** [comment]
@cckozie I finally figured out that as you said, the asterisk moving to the next line, no matter how big the window is, was due to the info icon next to it, not rendering properly the first time opened, but did so all other times. I found the easiest (and only) fix was to move the info icon to the end. 

I also fixed the language fields getting off if not populated.

**Comment by PhotoNomad0** [comment]
Verified in https://tc-builds.door43.org/build/develop/translationCore-macos-x64-1.1.0-3306aa7.dmg

**Comment by cckozie** [comment]
Looks good in  1.1.0 (3e1d1ee)
